Pipfile

```diff
@@ -4,15 +4,16 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-faust = "*"
 requests = "*"
 pyyaml = "*"
 thoth-storages = "*"
 thoth-common = "*"
 thoth-python = "*"
-thoth-messaging = "==0.7.13"
 jsonpath-ng = "*"
 prometheus-client = "*"
+thoth-messaging = "*"
+click = "*"
+typing-extensions = "*"
```

Comment:
I don't see this package as a direct dependency in sources. If Pipenv is causing trouble with the installation process, you need to remove the virtual environment+Pipfile.lock and let Pipenv relock the dependency tree once again in a fresh environment.